docs/docs_cli.rst

```diff
@@ -99,9 +99,9 @@ for a full setup.
     Try 'dwave config create --full' for more options.
 
     Creating new configuration file: /home/jane/.config/dwave/dwave.conf
-    Profile [defaults]:
-    Updating existing profile: defaults
-    Authentication token [skip]: ABC-1234567890abcdef1234567890abcdef
+    Profile [defaults]: ↵
+    Updating existing profile: defaults ↵
```

Comment:
This is not a prompt, just info output.

Reply:
some overenthusiasm on my part. Removed in both places